forte/processors/allennlp_processors.py

```diff
@@ -153,3 +165,39 @@ def _create_dependencies(input_pack, tokens, result):
                                   parent=tokens[heads[i] - 1],
                                   child=token)
             relation.rel_type = deps[i]
+
+    @staticmethod
+    def _create_srl(input_pack, tokens, result):
```

Comment:
Let's add typing to the variables and the return types. I know the existing functions don't have these. But we are enforcing it a bit more strict now.

It would be great if you also add the typing for those functions.

Reply:
Sure, will do